internal/configsource/builder_test.go

```diff
@@ -38,11 +37,11 @@ func TestConfigSourceBuild(t *testing.T) {
 	}
 
 	tests := []struct {
-		name               string
 		configSettings     map[string]ConfigSettings
 		factories          Factories
 		expectedCfgSources map[string]ConfigSource
 		wantErr            error
+		name               string
```

Comment:
I got errors like `fieldalignment: struct with 56 pointer bytes could be 48 (govet)` for structs like this one that's why the strange ordering.

Reply:
It would be nice if we had a formatter make target that runs the necessary fieldalignment commands.